src/packagedcode/pypi.py

```diff
@@ -83,7 +83,7 @@ def logger_debug(*args):
 
 @attr.s()
 class PythonPackage(models.Package):
-    metafiles = ('metadata.json', '*setup.py', 'PKG-INFO', '*.whl', '*.egg')
+    metafiles = ('metadata.json', '*setup.py', 'PKG-INFO', '*.whl', '*.egg', '*requirements*.txt', '*requirements*.in')
```

Comment:
Do we have test cases for these filename recognitions yet? if not we should add them

Reply:
https://github.com/nexB/scancode-toolkit/blob/83f29f9dbbca54a5d69db0d6d7617b8b4dd7021a/tests/packagedcode/test_pypi.py#L314-L333

I have added test here. I think this will be OK :)

Reply:
nice